DESCRIPTION

```diff
@@ -9,7 +9,6 @@ Depends:
     ccao,
     conflicted,
     dplyr,
-    embed,
```

Comment:
We dropped this package since it has extremely heavy dependencies. Instead we dummify categorical predictors.